.travis.yml

```diff
@@ -1,7 +1,20 @@
 sudo: required
+language: node_js
```

Comment:
This comes from the basic configuration for Cypress: https://docs.cypress.io/guides/guides/continuous-integration.html#Travis